E-Commerce/readme.md

```diff
@@ -46,16 +46,87 @@ erDiagram
 Write SQL queries to complete the following tasks:
 
 - [ ] List all the products whose name contains the word "socks"
+
+```sql
+SELECT * FROM products WHERE product_name ILIKE '%socks%';
+```
+
 - [ ] List all the products which cost more than 100 showing product id, name, unit price, and supplier id
+
+```sql
+SELECT p.id, p.product_name, pa.unit_price, PA.SUPP_ID
+    FROM products p INNER JOIN
+    product_availability pa on (p.id = pa.prod_id)
+    WHERE pa.unit_price > 100;
+```
+
 - [ ] List the 5 most expensive products
+
+```sql
+SELECT p.id, p.product_name, pa.unit_price, PA.SUPP_ID
+    FROM products p INNER JOIN
+    product_availability pa on (p.id = pa.prod_id)
+    ORDER BY pa.unit_price DESC
+    LIMIT 5;
+```
+
 - [ ] List all the products sold by suppliers based in the United Kingdom. The result should only contain the columns product_name and supplier_name
-- [ ] List all orders, including order items, from customer named Hope Crosby
+
+```sql
+SELECT p.product_name, s.supplier_name
+    FROM products p INNER JOIN
+    product_availability pa on (p.id = pa.prod_id)
+    INNER JOIN suppliers s on (pa.supp_id = s.id)
+    WHERE s.country = 'United Kingdom';
+```
+
+- [ ] List all orders, including order items, from customer named Hope Crosby.
+
+```sql
+SELECT *
+    FROM orders o INNER JOIN
+    order_items oi on (o.id = oi.order_id)
+    INNER JOIN customers c on (o.customer_id = c.id)
+    WHERE c.name ILIKE '%Hope Crosby%';
+```
+
 - [ ] List all the products in the order ORD006. The result should only contain the columns product_name, unit_price, and quantity
+
+```sql
+SELECT * FROM orders o
+    LEFT JOIN order_items oi ON (o.id = oi.order_id)
+    LEFT JOIN products p on (p.id = oi.product_id)
+    INNER JOIN product_availability pa on (p.id = pa.prod_id and oi.supplier_id = pa.supp_id)
+    WHERE order_reference = 'ORD006';
+
+
+SELECT p.product_name, pa.unit_price, oi.quantity
+    FROM products p LEFT JOIN
+    product_availability pa ON (p.id = pa.prod_id)
+    LEFT JOIN order_items oi ON (oi.product_id = pa.prod_id and oi.supplier_id = pa.supp_id)
+    LEFT JOIN orders o ON (o.id = oi.order_id)
+    WHERE o.order_reference = 'ORD006';
```

Comment:
CORRECT - as you demonstrate, there is more than one way to construct this query - well done!